src/navigation/py_slam/py_slam/node_sbg_slam.py

```diff
@@ -80,7 +81,7 @@ def nav_callback(self, ekf_nav_msg: SbgEkfNav, ekf_euler_msg: SbgEkfEuler):
         if not self.initial_pos and not self.initial_ang:
             coords: UTMPoint = fromLatLong(ekf_nav_msg.latitude, ekf_nav_msg.longitude, ekf_nav_msg.altitude)
             self.initial_pos = (coords.easting, coords.northing)
-            self.initial_ang = ekf_euler_msg.angle.z
+            self.initial_ang = ekf_euler_msg.angle.z - pi
```

Comment:
Are we going to merge this - pi hotfix, or going to wait until Harl checks out the SBG settings for initial heading?

Also applies to lines 95 & 142